lib/cdo/i18n_string_url_tracker.rb

```diff
@@ -94,6 +90,31 @@ def log(url, source, string_key, scope = [], separator = I18n.default_separator)
     add_to_buffer(normalized_key, logged_url, source, string_key.to_s, stringified_scope, separator)
   end
 
+  def self.string_key_exists?(string_key, scope = nil)
```

Comment:
Does the `translation_service.translated?` function serve the same purpose as this?

Reply:
We can combine the 2 functions in another PR (looks like they expect different input formats, specifically for the `scope` param). For now, I'll make the `string_exists?` function more generic so it can be used in all locales, not just the `I18n.default_locale`.